### PR TITLE
fix: Improve labels in ontology for display in App (DEV-3535)

### PR DESCRIPTION
**Pull request by BalduinLandolt**
### Task Description/Number

Issue Number: DEV-3535

The labels in the ontology are used as the property labels in the DSP-APP.

The current lables don't look good (cf. image below).

Also, labels need to be provided in all languages wherever we expect them to be displayed in the app.

image

### PR Type

- [ ] build/chore: maintenance tasks (no production code change)
- [ ] docs: documentation changes (no production code change)
- [ ] feat: represents new features
- [ ] fix: represents bug fixes
- [ ] perf: performance improvements
- [ ] refactor: represents production code refactoring
- [ ] test: adding or refactoring tests (no production code change)
- [ ] deprecated: Deprecation warning (ideally referencing a migration guide)

### Basic requirements for bug fixes and features

- [ ] Tests for the changes have been added
- [ ] Docs have been added / updated

### Does this PR introduce a breaking change?

- [ ] Yes

### Does this PR change client-test-data?

- [ ] Yes

### CODE REVIEW

File: webapi/src/main/resources/knora-ontologies/knora-base.ttl
Comment (editio) [inline webapi/src/main/resources/knora-ontologies/knora-base.ttl:1]: :hasTitle
    "Titolo"@it
:hasDescription
    "Descrizione"@it
:hasKeyword	
    "Parole chiave"@it
:relatesTo
    "Risorse correlate"@it
:isSegmentOf
    "Segmento di"@it	
:isAudioSegmentOf
    "Segmento di (Audio)"@it
:isVideoSegmentOf
    "Segmento di (Video)"@it
:subjectClassConstraint
    "Limiti del segmento"@it

Comment (flavens) [reply webapi/src/main/resources/knora-ontologies/knora-base.ttl:1]: :hasTitle
"Titre"@fr
:hasDescription
"Description"@fr
:hasKeyword
"Mots clés"@fr
:relatesTo
"Ressources associées"@fr
:isSegmentOf
"Segment de"@fr
:isAudioSegmentOf
"Segment de (Audio)"@fr
:isVideoSegmentOf
"Segment de (Vidéo)"@fr
:subjectClassConstraint
"Limites du segment"@fr

File: test_data/generated_test_data/ontologyR2RV2/knoraApiOntologyWithValueObjects.jsonld [hunk 1]
```diff
@@ -8576,7 +8576,7 @@
             "@id": "knora-api:isAnnotationOfValue"
         },
         {
-            "rdfs:label": "is audio segment of",
+            "rdfs:label": "Segment of (Audio)",
```
Comment (mpro7) [inline test_data/generated_test_data/ontologyR2RV2/knoraApiOntologyWithValueObjects.jsonld:8579]: What's the reason that type of the segment is in brackets?

**Review by mpro7 — APPROVED** [review]

File: webapi/src/main/resources/knora-ontologies/knora-base.ttl
Comment (seakayone) [reply webapi/src/main/resources/knora-ontologies/knora-base.ttl:1]: Thanks.

**Review by siers — APPROVED** [review]

File: test_data/generated_test_data/ontologyR2RV2/knoraApiOntologyWithValueObjects.jsonld (hunk 1 above)
Comment (mpro7) [reply test_data/generated_test_data/ontologyR2RV2/knoraApiOntologyWithValueObjects.jsonld:8579]: Thanks for the explanation. Yes, sorry, I didn't mention it in the previous comment. W/o brackets it looks for me more naturally. I would even say that the brackets actually highlight the "Audio" part even more ;)

Comment (BalduinLandolt) [reply test_data/generated_test_data/ontologyR2RV2/knoraApiOntologyWithValueObjects.jsonld]: I removed that part entirely - it should always be clear what type the target has anyway, so the information is redundant.

**Pull request merged**